shared/templates/file_owner/tests/missing_file_test.pass.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 #
 
-{{% if MISSING_FILE_PASS %}}
-    rm -f {{{ FILEPATH }}}
-{{% else %}}
-    true
-{{% endif %}}
+{{% for path in FILEPATH %}}
+    {{% if MISSING_FILE_PASS %}}
+        rm -f {{{ path }}}
+    {{% else %}}
+        mkdir -p $(basename {{{ path }}})
+        touch {{{ path }}}
+        chown {{{ FILEUID }}} {{{ path }}}
+    {{% endif %}}
+{{% endfor %}}
```

Comment:
```suggestion
{{% for path in FILEPATH %}}
    {{% if MISSING_FILE_PASS %}}
        rm -f {{{ path }}}
    {{% else %}}
        {{% if IS_DIRECTORY and FILE_REGEX %}}
        echo "Create specific tests for this rule because of regex"
        {{% elif IS_DIRECTORY and RECURSIVE %}}
        find -L {{{ path }}} -type d -exec chown {{{ FILEUID }}} {} \;
        {{% else %}}
        if [ ! -f {{{ path }}} ]; then
            mkdir -p "$(dirname '{{{ path }}}')"
            touch {{{ path }}}
        fi
        chown {{{ FILEUID }}} {{{ path }}}
        {{% endif %}}
    {{% endif %}}
{{% endfor %}}
```

let's use a similar implementation here as in: https://github.com/ComplianceAsCode/content/pull/7791/files#diff-d76b08eff4f123c6801e4e93d92db42e6296f1abac1e7fb1ab176c46b3dc36bc